src/google/protobuf/compiler/java/java_message_builder.cc

```diff
@@ -438,6 +450,62 @@ GenerateCommonBuilderMethods(io::Printer* printer) {
     "\n",
     "classname", name_resolver_->GetImmutableClassName(descriptor_));
 
+  printer->Print(
```

Comment:
I will ifdef that out in internal code base, but I don't think code size is a big issue here. In this beta-4 release we will also generate equals() and hashCode() methods for all proto files. That has a much larger impact on code size compared to this change but yet we do not seem to have a problem with it in our internal roll-out.